app/views/posts/show.html.erb

```diff
@@ -1,12 +1,23 @@
-<div class="detail full-width">
+<div class="detail full-width d-margin">
     <div class="post-head">
       <h3>Post  #<%= @post.id %> by <%= @user.name %></h3>
       <span>Comments: <%= @post.comments_counter || 0 %> Likes:<%= @post.likes_counter || 0 %></span>
     </div>
     <p class="comments"><%= @post.text%></p>
 </div>
-<div class="detail full-width d-margin">
+
+<div class="detail full-width like-comment">
+  <%= link_to new_comment_path(@post.id) do %>
+    <button>Add New Comment</button>
+  <% end %>
```

Comment:
![image](https://github.com/porag-m06/The-Ruby-on-Rails-Blog-App/assets/78251927/dd955e85-f924-474f-8ed4-5e7a834296a0)

Please ensure that it is possible for a user to add likes to posts. 😊 currently the button is not functional.

Kindly fix that 🛠👍

Reply:
![image](https://github.com/porag-m06/The-Ruby-on-Rails-Blog-App/assets/15729467/545cb2d0-f6bf-4c64-ba26-63e59ac7e81a)

Fixed now...